server/views/index.tmpl.html

```diff
@@ -26,6 +26,16 @@
 
 <body>
   <div id="mountNode"></div>
+  <script type="text/javascript">
+    var appInsights=window.appInsights||function(config){
```

Comment:
Also, I'd rather not link to a third party JS source, can we pull the client code in somehow?